src/app/entity/entity-mapper.service.ts

```diff
@@ -48,6 +48,29 @@ export class EntityMapperService {
     );
   }
 
+  /**
+   * Loads a list of Entity from the database whose IDs contain the prefix of the given resultEntity class.
+   *
+   * @param entityType the entity's class type (this would just be <code>Entity</code> for a pure entity).
+   * @returns A Promise containing an array with the entities.
+   */
+  public loadAll<T extends Entity>(entityType: { new(id: string): T; }): Promise<T[]> {
```

Comment:
Okay it's done. Let me know your opinion on this and then I'll adapt the test cases.